.changeset/many-rules-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': patch
+---
+
+put max-width on "Preview not available" text to guarantee wrapping
```

Comment:
@gbassham Could you move this as a bullet under the first changeset? Every PR should only have one changeset associated with it.